### PR TITLE
RDAPI-25990 New guide for configuring API Gateyway in EMT

### PR DESCRIPTION
**Pull request by credmond-axway**
Added guide for configuring an API Gateyway in EMT

## Describe the changes

This new page describes how to configure an API Gateyway in EMT mode, without the need to bake that configuration into a Docker image. Sample configuration scenarios are detailed.

## Deploy preview link

https://deploy-preview-2212--axway-open-docs.netlify.app/docs/apim_howto_guides/configuring_apigw_container/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+    "title": "Configuring an API Gateway Docker container",
```
Comment (andreamussap) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:2]: "Configure" - We write the headings in present tense.

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "linkTitle": "Configuring an API Gateway Docker container",
+    "weight": "30",
+    "date": "2022-02-01",
+    "description": "A guide on how to configure an API Gateway Docker container without the need to bake into a new Docker image."
+}
+
+## Introduction
```
Comment (andreamussap) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:9]: Unnecessary, delete it.

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+This document describes how an API Gateway, running in EMT mode, can be configured without the need to bake that configuration into a new Docker image. This process applies to API Gateway configurations only such as policy management, system property changes, environment settings, logging configurations etc.
+
+## Configuring an API Gateway Docker container
```
Comment (andreamussap) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:13]: Suggestion:
**Configure an API Gateway Docker container with Docker volumes**

Docker volumes make it easier to mount API Gateway configuration file (just one file? what's the name of the file?) to the Docker container, and make the file available to the API Gateway at startup time.

API Gateway configuration files are located in the host file system, and they must be mounted to the following specific mount points:

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md (hunk 1 above)
Comment (credmond-axway) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md]: Thanks. Updated.

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md (hunk 3 above)
Comment (credmond-axway) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md]: Thanks. I've re-worded this section.

**Review by jmbermingham — APPROVED** [review]

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    "title": "Create an API Gateway with Docker volumes",
+    "linkTitle": "Create an API Gateway with Docker volumes",
+    "weight": "30",
+    "date": "2022-02-01",
+    "description": "Configure an API Gateway Docker container to use a persisted volume for API Gateway configuration."
+}
+
+The primary documentation for [API Gateway in containers](/docs/apim_installation/apigw_containers/docker_script_gwimage) outlines the creation of images and their deployment to containers. This page describes how during the deployment of an image, you can use Docker volumes in a persisted store to update the configuration within the container. The main advantage of using Docker volumes is to allow you to update your configuration without having to rebake the Docker image.
+
+The major steps to create and configure your API Gateway in containers with Docker volumes are:
```
Comment (credmond-axway) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:11]: I would prefer to be specific about the use of docker in this scenario. Suggest we change to the following.

The major steps to create and configure your API Gateway in containers with Docker volumes are:

1. Mount the new API Gateway configuration.
2. Start the API Gateway docker container.
3. Verify the container configuration.

Comment (andreamussap) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md:11]: Sure! You're the expert. I'll update it now. Thanks

**Pull request merged**